### DISCUSSION TITLE
Truffle init not working in VS Code

### DISCUSSION THREAD

**Discussion by sakshijain009** [ConsenSys-archive/truffle#5146] [Installation]
- [ ] I've [opened a support ticket](https://trufflesuite.zendesk.com/hc/en-us/requests/new) before filing this issue.

---------------------------

## Issue

I installed truffle globally using the command

`npm install -g truffle@5.4.29`

But when i do 

`truffle init`

It is showing error.

## Expected Behavior

Should bootstrap the related folders.

## Actual Results

![Screenshot (153)](https://user-images.githubusercontent.com/62885857/170182978-319a4946-63a2-4ab4-93b3-106b9618ac8f.png)

![Screenshot (155)](https://user-images.githubusercontent.com/62885857/170183202-b918953f-346c-4ef3-88d7-36d28acf5600.png)

## Environment

* Operating System: Windows 10
* Ethereum client:
* Truffle version (`truffle version`): 5.4.29
* node version (`node --version`): 16.15.0
* npm version (`npm --version`): 8.5.5

**Comment by apdan510** [comment]
@sakshijain009 try this -

1. Click on the Windows Start button and search for "Powershell" and click on "Powershell - Run as Administrator"
![image](https://user-images.githubusercontent.com/40121410/170769469-a4000a14-a6fb-4d22-a141-3bd940b32a1e.png)

2. Enter this command into your Powershell terminal and enter the "A" key to accept: `Set-ExecutionPolicy RemoteSigned`
![image](https://user-images.githubusercontent.com/40121410/170769977-6be2cf1b-f6f7-4534-b702-d5f3b9048c08.png)

3. Try testing Truffle again with: `truffle version`
![image](https://user-images.githubusercontent.com/40121410/170770207-0c5fdf41-05be-4619-b38e-6a3386ab808b.png)
